intranet/static/js/eighth/schedule.js

```diff
@@ -37,110 +38,105 @@ $(function() {
             $(".propagate > i").removeClass(classes[0]).removeClass(classes[1])
                                .addClass(classes[2]);
         }
     }
 
     window.updatePropagateMethod = function(opt) {
-        console.info("New propagate method:", opt);
         window.propagate_method = opt;
     }
 
     $(".propagate").click(function() {
         var field = $(this).attr("data-field");
         var input = $(this).attr("data-input");
         var el = $("#" + input);
-        console.debug("Propagate", field, input, el.length > 0);
 
         var rows = $(".schedule-activity-grid tr.form-row");
-        if(window.propagate_direction && window.propagate_direction == "down") {
-            var current_row = el.parent().parent();
-            var rows = [];
-            var row = current_row;
-            while(row && row.length > 0) {
+        var row;
+        var current_row;
+        if (window.propagate_direction && window.propagate_direction === "down") {
+            current_row = el.parent().parent();
+            rows = [];
+            row = current_row;
+            while (row && row.length > 0) {
                 row = row.next();
-                if(row.hasClass("form-row")) {
+                if (row.hasClass("form-row")) {
                     rows.push(row[0]);
                 }
             }
             rows = $(rows);
         }
 
-        if(window.propagate_direction && window.propagate_direction == "up") {
-            var current_row = el.parent().parent();
-            var rows = [];
-            var row = current_row;
-            while(row && row.length > 0) {
+        if (window.propagate_direction && window.propagate_direction === "up") {
+            current_row = el.parent().parent();
+            rows = [];
+            row = current_row;
+            while (row && row.length > 0) {
                 row = row.prev();
-                if(row.hasClass("form-row")) {
+                if (row.hasClass("form-row")) {
                     rows.push(row[0]);
                 }
             }
             rows = $(rows);
         }
 
-        if(window.propagate_method && window.propagate_method == "only_new") {
+        if (window.propagate_method && window.propagate_method === "only_new") {
             var new_rows = [];
             for(var i=0; i<rows.length; i++) {
                 var rw = $(rows[i]);
-                if(!rw.hasClass("scheduled")) {
+                if (!rw.hasClass("scheduled")) {
                     new_rows.push(rows[i]);
                 }
             }
-            console.log(rows);
             rows = $(new_rows);
-            console.log(rows);
         }
 
-        if(el.hasClass("selectized")) {
-            console.info("Selectized element");
+        if (el.hasClass("selectized")) {
             var sel = el[0].selectize;
-            mod_items = sel.items;
-            console.info("New value:", mod_items);
+            var mod_items = sel.items;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'mod_items' is not in camel case.](https://www.codacy.com/app/tjhsst-ion/ion/file/2652533480/issues/source?bid=3212528&fileBranchId=3299315#l128)